.github/ISSUE_TEMPLATE.md

```diff
@@ -1,33 +1,32 @@
 <!--
-Before filing an issue, please check our Frequently Asked Questions:
+Before filing an issue, please verify the issue still occurs with the latest version of Jib.
 
-    https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md
+If you are encountering errors pulling or pushing to remote registries,
+please check our Frequently Asked Questions before filing an issue:
+
+    https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#registry-errors
 
 Please use this template to provide more context for your issue.
 -->
 
-**Description of the issue**:
+**Environment**:
 
-<!--
-If you are encountering errors pulling or pushing to remote registries,
-please check our FAQ before filing an issue.  Specifically:
+- *Jib version:*
+- *Build tool:* <!-- please include the version -->
```

Comment:
Isn't version specified above?

Reply:
This is supposed to be the Maven or Gradle version; I'll make that more explicit.